frontend/public/components/nav/items.tsx

```diff
@@ -307,7 +307,11 @@ export const PluginNavItems: React.FC<PluginNavItemsProps> = ({ items }) => {
         if (isSeparator(item)) {
           return <NavItemSeparator key={`separator-${index}`} />;
         }
-        return <li key={item.uid}>{createLink(item, true)}</li>;
+        return (
+          <li key={item.uid} className="dynamic-plugin__nav-item">
```

Comment:
It's better to give this a `co-` prefix to avoid ever conflicting with a plugin.

The BEM naming here isn't quite right. There's no `dynamic-plugin` block. I'd suggest just `co-nav-item`. It's unfortunate we can't use the PF class. Is there not a way to add the PF class to the parent `ul` if needed?

Reply:
The issue is there is not a parent `ul` that wraps the dynamic plugin nav links and i'm unable to figure out how to add one the way it's currently constructed.   

The original bug issue are the list bullets displayed and that is because the current markup output has an extra `li` around each link. So initially I changed `<div class="oc-perspective-nav">` to a `<ul className="oc-perspective-nav pf-c-nav__list">`  added the `pf-c-nav__item` class to ` <li key={item.uid}>{createLink(item, true)}</li>`  which removed the bullets but it never passed the e2e-gcp-console tests. 

The current changes add the new class `dynamic-plugin__nav-item` to the `li` and are removed with css.

```
// http://localhost:9000/dynamic-route-1

<div class="oc-perspective-nav">
  <li class="dynamic-plugin__nav-item">
    <li class="pf-c-nav__item pf-m-current">
      <a class="pf-c-nav__link pf-m-current pf-c-nav__link pf-m-current">Dynamic Nav 1</a>
    </li>
  </li>
  <li class="dynamic-plugin__nav-item">
    <li class="pf-c-nav__item">
      <a class="pf-c-nav__link pf-c-nav__link" href="/dynamic-route-2">Dynamic Nav 2</a>
    </li>
  </li>
  <li class="dynamic-plugin__nav-item">
    <li class="pf-c-nav__item">
      <a class="pf-c-nav__link pf-c-nav__link" href="/test-consumer">Test Consumer</a>
    </li>
  </li>
  <li class="dynamic-plugin__nav-item">
    <li class="pf-c-nav__item">
      <a class="pf-c-nav__link pf-c-nav__link" href="/test-utility-consumer">Test Utilities</a>
    </li>
  </li>
</div>
```

Reply:
Is there a reason not to use the PatternFly react controls? Just wondering.
```
<Nav onSelect={this.onNavSelect} aria-label="Nav">
    <NavList>
        <NavItem itemId={0} isActive={activeItem === 0} to="#">
```

Reply:
Converted to `div` so that extra `li` node wouldn't create bullet within nav list